packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -160,7 +160,7 @@ export class EditorPage {
 		// it is a fairly good stand-in.
 		await Promise.all( [
 			this.page.waitForURL( /(post|page|post-new.php)/ ),
-			this.page.waitForResponse( /.*posts.*/, { timeout: timeout } ),
+			this.page.waitForResponse( /post-new.php/, { timeout: timeout } ),
```

Comment:
On both `post` and `page` editor types, there is a request to the `<site_url>/wp-admin/post-new.php` endpoint.

The only difference is that for `page` type editors there is an additional query param `?post_type=page`.